src/components/subscriptions/data/hooks.js

```diff
@@ -117,7 +120,7 @@ export const useAllSubscriptionUsers = ({
     setLoading(true);
     LicenseManagerApiService.fetchSubscriptionUsers(
       subscriptionUUID,
-      { status: licenseStatusByTab[TAB_ALL_USERS], page_size: 500 },
+      { active_only: 1, page_size: 10000 },
```

Comment:
I was losing my mind because I couldn't find any of the bit you're removing on master. You're likely going to hit a hell of a lot of merge conflicts here when the base branch diffs against master.

I'm also unclear if we want to get rid of this tabbing experience. Unclear on that part exactly.

Reply:
I mean, we're going to merge the whole feature; no one working on bulk enrollment is commiting to master right now.

So, the reason this is a different hook from the one below it, is because the original was designed for use by the license management page, which has tabs by user. While I was developing the stepper, I just grabbed all the users. But I don't actually want that, I want just active users, and I want to put them all in the same table, not have them separated into tabs. Hence the separate hook.

Reply:
ah ah my bad. I was thinking the `useAllSubscriptionUsers` method existed at some point and your other branch was behind. I didn't realize it was added to that branch and that is what you are now changing.

Reply:
With that, and thus looking at this more clearly, the only issue I see here is similar to the comment Alex D had on your backend pr: is this page size an appropriate jump? Do we even currently have any cases where we have 500 records even to determine if there is a need for a larger result set?

Reply:
Switching to backend pagination, it turns out we do have customers with huge numbers of learners.